assets/scss/app.scss

```diff
@@ -27,6 +27,7 @@ p a[href] {
 .alert-complete {
     background-color: #00703c;
     border-color: #00703c;
+    padding: 50px !important;
```

Comment:
this could be a problem on mobile devices, could you please replace the markup for the corresponding section on the `check-requested.html` template according to GDS. in this case the `.alert-complete` can be removed as redundant.
here is an example markup:
https://design-system.service.gov.uk/patterns/confirmation-pages/

```
<div class="govuk-panel govuk-panel--confirmation">
          <h1 class="govuk-panel__title">
            Application complete
          </h1>
          <div class="govuk-panel__body">
            Your reference number<br><strong>HDJ2123F</strong>
          </div>
        </div>
```

Reply:
Done! :)